tools/grid/test/grid.test.ts

```diff
@@ -51,7 +51,8 @@ describe('Grid', () => {
 
         expect(el.tabIndex).to.equal(0);
 
-        el.focus();
+        await sendKeys({ press: 'Tab' });
+        await sendKeys({ press: 'Tab' });
 
```

Comment:
Can we have an assertion or comment here to understand what these Tabs are doing?

Reply:
Instead of calling focus() function, we are focusing using Tab key

Reply:
can we add an assertion to check if the element is indeed focused?

Reply:
The code after line 56 is checking for exactly the same!